diff -Nru kvpnc-0.8.4.orig/admin/config.sub kvpnc-0.8.4/admin/config.sub
--- kvpnc-0.8.4.orig/admin/config.sub	2006-04-09 15:17:23.102983750 +0000
+++ kvpnc-0.8.4/admin/config.sub	2006-04-09 15:17:45.512384250 +0000
@@ -280,6 +280,9 @@
 	i*86 | x86_64)
 	  basic_machine=$basic_machine-pc
 	  ;;
+	amd64-*)
+	  basic_machine=x86_64-`echo $basic_machine | sed 's/^[^-]*-//'`
+	  ;;
 	# Object if more than one company name word.
 	*-*-*)
 		echo Invalid configuration \`$1\': machine \`$basic_machine\' not recognized 1>&2
